crates/release_plz_core/src/command/release_pr.rs

```diff
@@ -62,6 +62,10 @@ impl ReleasePrRequest {
         self
     }
 
+    pub fn branch_prefix(&self) -> &str {
+        self.branch_prefix.as_str()
```

Comment:
```suggestion
        &self.branch_prefix
```
nit: more concise :)